### PR TITLE
Chore: Clean up old FS custom events / identification code 

### PR DESCRIPTION
**Pull request by lxhughes**
Asana: https://app.asana.com/0/home/50959463118296/1202286302110707

### CODE REVIEW

File: assets/tests/contexts/notificationsContext.test.tsx [hunk 1]
```diff
@@ -1,7 +1,5 @@
 import { act as hooksAct, renderHook } from "@testing-library/react-hooks"
-import { mount } from "enzyme"
```
Comment (lemald) [inline assets/tests/contexts/notificationsContext.test.tsx:2]: Hooray for getting rid of old Enzyme tests!

**Review by lemald — CHANGES_REQUESTED** [review]
One relatively minor thing that can be changed, but overall looks good to me!

File: test/skate_web/views/layout_view_test.exs [hunk 2]
```diff
@@ -1,26 +1,4 @@
...
-
-      refute LayoutView.record_fullstory?()
-    end
-
-    test "returns false if the :record_fullstory env is missing" do
-      reassign_env(:skate, :record_fullstory, nil)
-
-      refute LayoutView.record_fullstory?()
-    end
-  end
```
[the remark this answers is not in the document]
Comment (lxhughes) [reply test/skate_web/views/layout_view_test.exs:25]: good point! removed in b960e881

**Comment by lemald** [comment]
Okay, this now looks good to me! I'm going to hold off on actually approving until we know we're ready to stop using FullStory, though.

**Review by lemald — APPROVED** [review]
👍

**Pull request merged**